tikzplotlib/line2d.py

```diff
@@ -31,6 +31,17 @@ def draw_line2d(data, obj):
     data, line_xcolor, _ = mycol.mpl_color2xcolor(data, color)
     addplot_options.append(line_xcolor)
 
+    # get draw style
+    drawstyle = obj.get_drawstyle()
+    if drawstyle is not (None or "default"):
```

Comment:
Better:
```python
if drawstyle in [None, "default"]:
    pass
elif ...
```
Also, a test would be nice. Not strictly required though, it's just that something could be destroyed in the future then.